internal/s/subtree.go

```diff
@@ -54,7 +52,7 @@ func (spec SupervisorSpec) run(
 		supTolerance,
 		supRscCleanup,
 		notifyCh,
-		ctrlCh,
+		ctrlChan,
```

Comment:
With the channel now being created further away from its use inside of `runMonitorLoop`, I think there needs to be some more thought into its cleanup. Is there an issue if it is never closed? Closing the channel might cause issues with more than one thing indicting that a shutdown is happening (the context and now the closed control channel), but I'm not familiar enough with channels to know if there are issues in not closing them. If it is closed, receiving on it will return `nil` because it carries an interface, right?

Reply:
I may be walking back my previous comment https://github.com/capatazlib/go-capataz/pull/66#discussion_r608306385